java/maven/src/org/netbeans/modules/maven/problems/MavenModelProblemsProvider.java

```diff
@@ -328,4 +392,58 @@ private String getDescriptionText(Throwable e) {
             return TXT_NoMsg(path);
         }
     }
+    
+    //-------------------------------------------------------------------------
+    // ActionProvider implementation
+    
+    private static final String[] PROBLEM_ACTIONS = { ActionProvider.COMMAND_PRIME };
+
+    @Override
+    public String[] getSupportedActions() {
+        return PROBLEM_ACTIONS;
```

Comment:
Please add a test to verify the action is present, possibly also that it behaves kind of expected.

Reply:
Tests added as 7d140d0; tried to test all 3 cases: 
- action enabled since problems were not identified yet, but the priming build itself won't run
- action not enabled on project analyzed to been OK
- action enabled on broken project, priming build relly executes